astro/src/content/docs/sdks/android-sdk.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: FusionAuth Android SDK
+description: The FusionAuth Android SDK allows you to add different OAuth2 and OpenID Connect functionalities to your Android application. It's a highly standardized and simplified starting point for developers to easily integrate FusionAuth in their own custom mobile apps by taking care of all the dependencies.
+navcategory: developer
+section: sdks
+disableTOC: true
+---
+import RemoteContent from 'src/components/RemoteContent.astro';
+
+## Overview
+
+<RemoteContent url="https://raw.githubusercontent.com/FusionAuth/fusionauth-android-sdk/main/README.md"
+    tags="forDocSiteOverview" />
+
+## Getting Started
+
+<RemoteContent url="https://raw.githubusercontent.com/FusionAuth/fusionauth-android-sdk/main/README.md"
```

Comment:
This is something we tested thoroughly, it's how to work locally with android emulator. And it's how the automated workflow in github works.

Reply:
We have the TESTING doc we're working on which deep dives in to E2E testing which makes use of the 10.0.2.2 IP and stuff like ngrok would then again cause issues with it. So to avoid confusion we will further explain what that IP means (its basically the host address from the perspective of the Android Emulator) and in what context this IP is important/used.